app/src/test/java/com/github/se/cyrcle/io/Settings.kt

```diff
@@ -0,0 +1,61 @@
+package com.github.se.cyrcle.io
+
+import java.io.InputStream
+import java.io.OutputStream
+
+data class Settings(
```

Comment:
Will the values that you'll actually put in later need a validation function or do we expect that to be done previously elsewhere ?

Reply:
What do you mean by *validation* ? Like checking is a number shouldn't be negative ? This would done at class instantiation (construction)
If you're speaking only for this class, you'll notice that this one is simply for tests, so there is no point in having data validation for it (other that for tests)